release_scripts/signassert.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2011-2015, Armory Technologies, Inc.                           #
+# Distributed under the GNU Affero General Public License (AGPL v3)            #
+# See LICENSE or http://www.gnu.org/licenses/agpl.html                         #
+#                                                                              #
+################################################################################
+
+import getpass
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+from armoryengine.ALL import *
```

Comment:
Okay. I did a little research and think that [this might work](http://stackoverflow.com/questions/15494574/using-pythons-argparse-in-multiple-scripts-backed-by-multiple-custom-modules). Can you give it a whirl? If it works, maybe the flags something obvious, like --ecdsa_signer or whatever seems appropriate.